src/screens/benefit/Details.tsx

```
@@ -224,6 +225,7 @@ const BenefitsDetails: React.FC = () => {
 						...(applicationData?.application_data || {}),
 						external_application_id:
 							applicationData?.external_application_id,
+							remark:applicationData?.remark
```

ESLint

[error] Expected indentation of 5 tabs but found 7.

(indent)